python3-Pillow.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "python3-Pillow",
```

Comment:
building this failed on arm. 
https://flathub.org/builds/#/builders/44/builds/172
I know that this works in other apps on flathub using a slightly different recipe: https://github.com/flathub/org.gabmus.hydrapaper/blob/68b9d617d937d3eb4137150e5e98546b41c7fb57/org.gabmus.hydrapaper.json#L31